src/models/Project/index.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import createProject from './createProject'
+export * from './createProject'
+
+export type Project = {
+  id: number,
```

Comment:
I think GraphQL returns `id` as `string` and not `number`, even if you pass it a number.

Reply:
Also fixed with 48a68a0